spec/features/users/user_signs_up_spec.rb

```diff
@@ -1,41 +1,41 @@
-# require 'rails_helper'
-#
-# feature 'user registers', %Q{
-#   As a visitor
-#   I want to register
-#   So that I can create an account
-# } do
-#
-#   # Acceptance Criteria:
-#   # * I must specify a valid email address,
-#   #   password, and password confirmation
-#   # * If I don't specify the required information, I am presented with
-#   #   an error message
-#
-#   scenario 'provide valid registration information' do
-#     visit new_user_registration_path
-#
-#     fill_in 'First Name', with: 'Jeff'
-#     fill_in 'Last Name', with: 'Sampson'
-#     fill_in 'Desired Username', with: 'sampsonite3000'
-#     fill_in 'Email', with: 'john@example.com'
-#     fill_in 'user_password', with: 'password'
-#     fill_in 'Password Confirmation', with: 'password'
-#
-#     click_button 'Sign up'
-#
-#     expect(page).to have_content('Welcome! You have signed up successfully.')
-#     expect(page).to have_content('Sign Out')
-#   end
-#
-#   scenario 'provide invalid registration information' do
-#     visit new_user_registration_path
-#
-#     click_button 'Sign up'
-#     expect(page).to have_content("First name can't be blank")
-#     expect(page).to have_content("Last name can't be blank")
-#     expect(page).to have_content("Email can't be blank")
-#     expect(page).to have_content("Password can't be blank")
-#     expect(page).to_not have_content('Sign Out')
-#   end
-# end
+require 'rails_helper'
+
+feature 'user registers', %Q{
+  As a visitor
+  I want to register
+  So that I can create an account
+} do
+
+# Acceptance Criteria:
+# * I must specify a valid email address,
+#   password, and password confirmation
+# * If I don't specify the required information, I am presented with
+#   an error message
+
+  scenario 'provide valid registration information' do
+    visit new_user_registration_path
+    within('#new-user') do
+      fill_in 'First Name', with: 'Jeff'
+      fill_in 'Last Name', with: 'Sampson'
+      fill_in 'Email', with: 'jeffsampson@example.com'
+      fill_in 'user_password', with: 'password'
+      fill_in 'Password Confirmation', with: 'password'
+
+      click_button 'Sign up'
+    end
+    expect(page).to have_content('Welcome! You have signed up successfully.')
+    expect(page).to have_content('Sign Out')
+  end
+
+  scenario 'provide invalid registration information' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.